app/helpers/dashboards_helper.rb

```diff
@@ -2,4 +2,20 @@ module DashboardsHelper
   def project_has_different_deploys?(deploy_group_versions)
     deploy_group_versions.values.map(&:reference).uniq.count > 1
   end
+
+  def project_style(project_id)
```

Comment:
this will be a global method in all templates, so something more meaningful / unique would be good